GSoC-2021/Kush-Kothari.md

```diff
@@ -0,0 +1,82 @@
+# GSoC 2021 Expression of Interest
+## Kush Kothari (Web and Python developer | ML Enthusiast)
+#### Email: kushkothari2001@gmail.com
+#### GitHub profile: [kkothari2001](https://github.com/kkothari2001)
+#### LinkedIn profile: [Kush Kothari](https://www.linkedin.com/in/kush-kothari-ba013218b/)
+
+
+## Assessment Step 1.
+
+Link to the deployed site: [ohuploaderkushk](https://ohuploaderkushk.herokuapp.com/)
+
+Link to activity page on OpenHumans: [GSoC assessment for Kush Kothari](https://www.openhumans.org/activity/gsoc-assesment-for-kush-kothari/)
+
+## Assessment Step 2.
+
+### Django Projects:
+- [To-Do App](https://github.com/kkothari2001/django-todo): App that can be used to maintain a list of tasks that can be dynamically added and deleted once completed.
```

Comment:
Thanks @kkothari2001 - do you have a deployment somewhere for me to test?

I note that the repository doesn't have a README, licence nor any installation instructions. I'll have a look for these in some of the other projects.

Reply:
I have added a deployment, license and a README to the repos.